content/geometry/object-human/human-model-structure.adoc

```diff
@@ -18,20 +19,33 @@ Skeleton
         |_ Upper_Arm_Right (T)
           |_ Lower_Arm_Right (T)
             |_ Full_Hand_Right (T)
+            |_ Hand_Right (T)
+              |_ Upper_Fingers_Right (T)
+                |_ Lower_Fingers_Right (T)
+              |_ Upper_Thumb_Right (T)
+                |_ Lower_Thumb_Right (T)
+            |_ Gripping_Point_Right (T)
       |_ Shoulder_Left (T)
         |_ Upper_Arm_Left (T)
           |_ Lower_Arm_Left (T)
             |_ Full_Hand_Left (T)
+            |_ Hand_Left (T)
+              |_ Upper_Fingers_Left (T)
+                |_ Lower_Fingers_Left (T)
+              |_ Upper_Thumb_Left (T)
+                |_ Lower_Thumb_Left (T)
+            |_ Gripping_Point_Left (T)
     |_ Upper_Leg_Right (T)
       |_ Lower_Leg_Right (T)
         |_ Full_Foot_Right (T)
+        |_ Standing_Point_Right (T)
     |_ Upper_Leg_Left (T)
       |_ Lower_Leg_Left (T)
         |_ Full_Foot_Left (T)
-Body
-Hair
-Clothing
-Grp_Equipment_Dynamic (T)
+        |_ Standing_Point_Left (T)
+  |_ Body
+  |_ Hair
+  |_ Clothing
```

Comment:
I think Body, Hair and Clothing should not be below the Skeleton in this structure. They are the meshes the Skeleton is linked to as far as I understand.